index.html

```diff
@@ -35,22 +35,22 @@
 
           <!-- you can put rows inside of columns! -->
           <!-- Bootstrap subdivides these 'nested' rows into 12 columns, too! -->
-          <div class="row 1">
-            <div id=a1 class="col-md-4 box"></div>
-            <div id=a2 class="col-md-4 box"></div>
-            <div id=a3 class="col-md-4 box"></div>
+          <div class="row-1">
+            <div id=a1 class="col-xs-4 box col-md-4 box col-lg-4 box"></div>
```

Comment:
It's best to include quotes around all attribute values, e.g. "id="a1". ;)

What would happened if you used a class-based approach, and didn't use ids at all?